internal/i18n/README.md

```diff
@@ -24,8 +24,38 @@ To update an existing language:
 
 1.  Find the language in the `locales/` folder.
 
+1.  Update the language on line 3.
+
 1.  Update the `msgstr` fields to the new values.
 
 1.  Save the file.
 
 1.  Submit a pull request.
+
+
+## Terms
+
+-   **realm** - public health authority, state, or hospital system. The system
+    is multi-tenant, and a user can be a member of multiple realms. For example,
+    the same doctor might operate clinics across state lines. In the US, each
+    state is a "realm". In other areas, a realm could be a country or region.
+
+-   **case worker** - someone at a health clinic, testing facility, doctor's
+    office, etc that is managing cases related to COVID-19.
+
+-   **patient** - the person who has tested positive or negative for COVID-19.
```

Comment:
not necessarily tested positive - the person being given test results?

Reply:
Done.